static/js/page.js

```diff
@@ -440,20 +440,17 @@ function displayTokens(j) {
 }
 
 function populateStats(stats) {
-   $("#tok-num").text(format_is(stats.num_tokens));
-   $("#num-sent").text(format_is(stats.num_sentences));
-   $("#num-parsed-sent").text(format_is(stats.num_parsed));
-   if (stats.num_parsed == 1) {
-      $("#paragraphs").text("málsgrein")
-   } else {
-      $("#paragraphs").text("málsgreinar");
-   }
-   if (stats.num_sentences > 0) {
-      $("#num-parsed-ratio").text(format_is(100.0 * stats.num_parsed / stats.num_sentences, 1));
-   } else {
-      $("#num-parsed-ratio").text("0.0");
-   }
-   $("#avg-ambig-factor").text(format_is(stats.ambiguity, 2));
+   var parsedRatio = stats.num_sentences > 0 ? format_is(100.0 * stats.num_parsed / stats.num_sentences, 1) : "0.0"
+   var statisticsSummary = $("#statistics-summary")
+   statisticsSummary.append(
```

Comment:
Might be safer to add `statisticsSummary.html("");` here

Reply:
Indeed. Fixed.